pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -134,7 +134,7 @@ type RequestedToCapacityRatioArgs struct {
 	Shape []UtilizationShapePoint
 	// Resources to be considered when scoring.
 	// The default resource set includes "cpu" and "memory" with an equal weight.
-	// Allowed weights go from 1 to 100.
+	// Allowed weights should be larger than 0.
```

Comment:
"Weights should be larger than 0" or "Allowed weights are larger than 0"

Reply:
Yes, this sounds better.